xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 set(HEADERS General.h
             ListItem.h
             Window.h
-            definitions.h)
+            definitions.h
+            renderHelper.h)
```

Comment:
I think it's quite unusual (if not violating our coding guidelines) to introduce source files starting with a lowercase letter.

Reply:
Here's a little special case for me, files that start with Uppercase are meant to be used directly from the addon. Those with lowercase are only headers that are used in the background by other headers, so first insignificant for addon developers.